packages/webawesome/src/components/dialog/dialog.ts

```diff
@@ -76,6 +77,13 @@ export default class WaDialog extends WebAwesomeElement {
   /** When enabled, the dialog will be closed when the user clicks outside of it. */
   @property({ attribute: 'light-dismiss', type: Boolean }) lightDismiss = false;
 
+  /** The ID of the element that labels the dialog.
+   *  Overrides the default `aria-labelledby="title"`. */
+  @property({ attribute: 'aria-labelledby' }) ariaLabelledby?: string;
+
+  /** The ID of the element that describes the dialog. */
+  @property({ attribute: 'aria-describedby' }) ariaDescribedby?: string;
```

Comment:
This is not a pattern we use in the library. All labels and descriptions are abstracted into `label` and `description` properties, oftentimes with corresponding slots for HTML content.

To the best of my knowledge, native dialogs don't require aria labels as the dialog is announced as a group, and the label is the first thing that gets read by screen readers due to its position.

An argument could be made for adding a `description` property, but I'm not sure that's necessary either as the dialog can contain arbitrary content and descriptions are more common on form controls.

To save yourself the work, please open an issue and ask before proposing API changes.